src/ucm/bistro/bistro.c

```diff
@@ -45,6 +48,59 @@ static ucs_status_t ucm_bistro_protect(void *addr, size_t len, int prot)
     return UCS_OK;
 }
 
+void ucm_bistro_apply_locking(void *dst, const ucm_bistro_lock_t *bytes)
```

Comment:
maybe rename to ucm_bistro_modofy_code? and move clear_cache inside this function?

Reply:
Fixed for the name, but for clarity I'd rather keep the three clear cache calls in the single function below. (Also the last change sequence is modify_code+protect_change+clear_cache, so adding clear cache to modify would make one more clear_cache call).

Reply:
ik